terraform/cloudfront/cloudfront_distribution/variables.tf

```diff
@@ -0,0 +1,17 @@
+variable "aliases" {
+  type = "list"
+}
+
+variable "comment" {}
+
+variable "env" {}
+
+variable "name" {}
+
+variable "origin" {}
+
+variable "system_dns_zone_id" {}
+
+variable "system_dns_zone_name" {}
+
+variable "system_domain_cert_arn" {}
```

Comment:
Sure! I don't think we've been doing that in many of the other files, but you're right.